Bad import in Breadcrumbs

I tried installing Cashmere into an Angular 8 application with Ivy enabled.  When trying to build, I get the following error:

```

ERROR in node_modules/@healthcatalyst/cashmere/lib/breadcrumbs/breadcrumbs.component.d.ts:3:37 
- error TS2307: Cannot find module '@angular/router/src/config'.

3 import { QueryParamsHandling } from '@angular/router/src/config';
                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```

It was probably an auto-import error (I've seen VSCode do this sometimes), so is probably an easy fix, but completely blocks usage of the library (even if BreadcrumbModule isn't imported) from any application using Angular 8 and Ivy :(

--- a/projects/cashmere/src/lib/breadcrumbs/breadcrumbs.component.ts
+++ b/projects/cashmere/src/lib/breadcrumbs/breadcrumbs.component.ts
@@ -1,6 +1,5 @@
-import { Component, OnDestroy, OnInit, Input } from '@angular/core';
-import { ActivatedRoute, Event, NavigationEnd, Params, PRIMARY_OUTLET, Router } from '@angular/router';
-import { QueryParamsHandling } from '@angular/router/src/config';
+import {Component, OnDestroy, OnInit, Input} from '@angular/core';
+import {ActivatedRoute, Event, NavigationEnd, Params, PRIMARY_OUTLET, Router} from '@angular/router';
 
 /**
  * IBreadcrumb interface is used to store all required data for each breadcrumb element
@@ -11,6 +10,12 @@ export interface IBreadcrumb {
     params?: Params;
     url: string;
 }
+
+/*
+  This type is from @angular/router, but the import location varies by Angular version
+  Including here to allow Cashmere to work regardless of Angular version
+*/
+export type QueryParamsHandling = 'merge' | 'preserve' | '';
 
 /** A navigational aid that allows users to keep track of their location within the current application  */
 @Component({
@@ -35,7 +40,7 @@ export class BreadcrumbsComponent implements OnInit, OnDestroy {
         this._queryParamsHandling = queryParamsHandling;
     }
 
-    constructor(private activatedRoute: ActivatedRoute, private router: Router) { }
+    constructor(private activatedRoute: ActivatedRoute, private router: Router) {}
 
     ngOnInit() {
         const ROUTE_DATA_BREADCRUMB: string = 'breadcrumb';

